clientv3/balancer.go

```diff
@@ -221,14 +248,62 @@ func (b *simpleBalancer) updateNotifyLoop() {
 
 func (b *simpleBalancer) notifyAddrs() {
 	b.mu.RLock()
-	addrs := b.addrs
+	eps := make([]addrConn, 0, len(b.addrs))
+	for _, ac := range b.addrs {
+		eps = append(eps, ac)
+	}
+	multi := len(b.addrs) > 1
```

Comment:
notifyAddrs isn't called very often, no need to skip the sort operation for a single address